test/Microsoft.Azure.WebJobs.Extensions.ServiceBus.Tests/ServiceBusEndToEndTests.cs

```diff
@@ -212,6 +228,65 @@ public async Task BindToString()
             Assert.Contains("Input(foobar)", logs);
         }
 
+        [Fact]
+        public async Task MessageDrainingQueue()
+        {
+            await TestSingleDrainMode<DrainModeValidationFunctions>(true);
+        }
+
+        [Fact]
+        public async Task MessageDrainingTopic()
+        {
+            await TestSingleDrainMode<DrainModeValidationFunctions>(false);
+        }
+
+        [Fact]
+        public async Task MessageDrainingQueueBatch()
+        {
+            await TestMultipleDrainMode<DrainModeValidationFunctions>(true);
+        }
+
+        [Fact]
+        public async Task MessageDrainingTopicBatch()
+        {
+            await TestMultipleDrainMode<DrainModeValidationFunctions>(false);
+        }
+
+        /*
+         * Helper functions
+         */
+
+        private async Task TestSingleDrainMode<T>(bool sendToQueue)
+        {
+            var host = BuildTestHost<DrainModeValidationFunctions>();
+            await host.StartAsync();
+
+            _drainValidationPreDelay = new ManualResetEvent(initialState: false);
+            _drainValidationPostDelay = new ManualResetEvent(initialState: false);
+
+            if (sendToQueue)
+            {
+                await WriteQueueMessage(_primaryConnectionString, FirstQueueName, "queue-message-draining-no-sessions-1");
+            }
+            else
+            {
+                await WriteTopicMessage(_primaryConnectionString, TopicName, "topic-message-draining-no-sessions-1");
```

Comment:
add "topic-message-draining-no-sessions-1" to constants

Reply:
Done.